hydra-node/src/Hydra/Options.hs

```diff
@@ -716,6 +726,27 @@ toArgs
       , contestationPeriod
       } = chainConfig
 
+defaultRunOptions :: RunOptions
+defaultRunOptions =
+  RunOptions
+    { verbosity = Verbose "run-options"
+    , nodeId = NodeId "hydra-node-1"
+    , host = localhost
+    , port = defaultPort
+    , peers = []
+    , apiHost = localhost
+    , apiPort = defaultPort
+    , monitoringPort = Nothing
+    , hydraSigningKey = "/tmp/hydra.sk"
+    , hydraVerificationKeys = []
+    , hydraScriptsTxId = generateWith arbitrary 42
+    , persistenceDir = "/tmp/persistence"
+    , chainConfig = defaultChainConfig
+    , ledgerConfig = defaultLedgerConfig
+    }
+ where
+  localhost = IPv4 $ toIPv4 [127, 0, 0, 1]
```

Comment:
These are not the defaults in use. Could you ensure they are the same as also used by the `parse...` functions? e.g. `Verbose "run-options"` should be `Verbose "HydraNode"`